src/main/java/xyz/aadev/generitech/common/network/messages/power/PacketSides.java

```diff
@@ -0,0 +1,101 @@
+package xyz.aadev.generitech.common.network.messages.power;
+
+import io.netty.buffer.ByteBuf;
+import net.minecraft.client.network.NetHandlerPlayClient;
+import net.minecraft.inventory.IInventory;
+import net.minecraft.item.ItemStack;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.item.ItemStack'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)